### DISCUSSION TITLE
Hide "This branch is X commits ahead of Y"

### DISCUSSION THREAD

**Discussion by kierandrewett** [community/community#23637] [New to GitHub]
<p>Is there a way to hide the branch commits ahead message? I maintain a fork of an existing project and I don’t plan to merge in the changes to upstream, it can get pretty annoying having that there.</p>
<p>Thanks.</p>

**Comment by ghost** [comment, marked as answer]
<p>Unfortunately, you can’t. :confused: Sorry about that</p>
